tests/unit/python/foglamp/services/core/scheduler/test_scheduler_entities.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test foglamp/services/core/scheduler/entities.py """
+
+import uuid
+import pytest
+from foglamp.services.core.scheduler.entities import *
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2018 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("services", "core", "scheduler")
+class TestSchedulerEntities:
+    def test_ScheduledProcess(self):
+        scheduled_process = ScheduledProcess()
+        assert scheduled_process.name is None
+        assert scheduled_process.script is None
+
+    def test_Schedule(self):
+        assert isinstance(Schedule.Type(2), IntEnum)
+
+        schedule = Schedule(Schedule.Type.STARTUP)
+        assert schedule.schedule_id is None
+        assert schedule.name  is None
+        assert schedule.exclusive  is True
+        assert schedule.enabled is False
+        assert schedule.repeat is None
+        assert schedule.process_name is None
+        assert schedule.schedule_type == 1
+
+    def test_StartUpSchedule(self):
+        schedule = Schedule(Schedule.Type.STARTUP)
+        startup_schedule = StartUpSchedule()
+        assert startup_schedule.schedule_id is None
+        assert startup_schedule.name  is None
+        assert startup_schedule.exclusive  is True
+        assert startup_schedule.enabled is False
+        assert startup_schedule.repeat is None
+        assert startup_schedule.process_name is None
+        assert startup_schedule.schedule_type == 1
+        with pytest.raises(AttributeError) as excinfo:
+            assert startup_schedule.day is None
+            assert startup_schedule.time is None
```

Comment:
These assertions are never called. If you replace `is None` with `is not None` test passes then too?